numba/tests/test_comprehension.py

```diff
@@ -248,14 +222,219 @@ def list24(x):
             bits = 64
         else:
             bits = 32
-        
+
         if utils.PYVERSION < (3, 0):
             with self.assertRaises(TypingError) as raises:
                 cfunc = jit(nopython=True)(list22)
                 cfunc(var)
             msg = "cannot unify reflected list(int%d) and int%d" % (bits, bits)
             self.assertIn(msg, str(raises.exception))
 
+class TestArrayComprehension(unittest.TestCase):
+
+    def check(self, pyfunc, *args, run_parallel=False, assert_allocate_list=False,
```

Comment:
This function signature does not work in Python 2.7.  `*args` can't be followed by explicit keyword args, only `**kwargs`.  Since this is just test code, it might be easier to pass the extra positional args as a list.